internal/auth0/client/resource_test.go

```diff
@@ -1060,6 +1060,42 @@ resource "auth0_client" "my_client" {
 }
 `
 
+const testAccUpdateClientWithSAMLP = `
+resource "auth0_client" "my_client" {
+	name = "Acceptance Test - SSO Integration - {{.testName}}"
+	app_type = "sso_integration"
+
+	addons {
+		samlp {
+			issuer                             = "https://tableau-server-test.domain.eu.com/api/v1"
+			audience                           = "https://tableau-server-test.domain.eu.com/audience-different"
+			destination                        = "https://tableau-server-test.domain.eu.com/destination"
+			digest_algorithm                   = "sha256"
+			lifetime_in_seconds                = 3600
+			name_identifier_format             = "urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
+			name_identifier_probes             = [ "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress" ]
+			create_upn_claim                   = false
+			passthrough_claims_with_no_mapping = false
+			map_unknown_claims_as_is           = false
+			map_identities                     = false
+			typed_attributes                   = false
+			recipient                          = "https://tableau-server-test.domain.eu.com/recipient-different"
+			signing_cert                       = "-----BEGIN PUBLIC KEY-----\nMIGf...bpP/t3\n+JGNGIRMj1hF1rnb6QIDAQAB\n-----END PUBLIC KEY-----\n"
```

Comment:
Should values for `authn_context_class_ref`, `binding`, `include_attribute_name_format`, `sign_response`, and `signature_algorithm` be included and then asserted?